samples/buck_pins/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022-2023 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2023 Nordic Semiconductor ASA
```

All new samples should be from the 2023 year.

Reply:
Bump